### PR TITLE
docs(core): add description in README for passing xml with the web component

### LINKED ISSUE #25: Pass default bpmn diagram in web component
I have a use case in which I have to pass default `bpmnDiagram`(XML) while editing the workflow. Which should be picked up by `workflow-creator` and render the initial data.

How do I pass default/initial value of `bpmnDiagram` in `workflow-creator`

**Additional context**
I am using web component in React.

**NOTE**
Same should work in angular library

### PR DESCRIPTION
**Pull request by arpit1503khanna**
GH-25

## Description
add the description in README about the usage of `bpmnDiagram`(XML) with web component.   

Fixes #25 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by yeshamavani** [comment]
rectify the commit message.

File: projects/workflows-creator/README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ npm i @sourceloop/workflows-creator
 ### Usage
 
 The web component accepts all the same inputs and services as the regular Angular Module, but instead of passing them through bindings and DI, you pass them as properties of the element as shown below.
-NOTE: If `envIdentifier` is required it must be provided in the global `window` object as shown below, otherwise it will be rendered as undefined. This step is needed only in non-angular applications. For angular appplications it can be directly provided with the help of providers.
+NOTE: If `envIdentifier` is required it must be provided in the global `window` object as shown below, otherwise it will be rendered as undefined. This step is needed only in non-angular applications. For angular appplications it can be directly provided with the help of providers. If the value of bpmn diagram is required it can passed to web component in the manner as described below. This can be used to dynamically use the XML when working with the sourceloop-workflow web component.
```
Comment (yeshamavani) [inline projects/workflows-creator/README.md:102]: specify that diagram attribute needs to be set 
justifying your added piece of code in example below

**Review by yeshamavani — CHANGES_REQUESTED** [review]
commit message needs to be rectified

**Comment by yeshamavani** [comment]
@arpit1503khanna  is this resolved

**Comment by arpit1503khanna** [comment]
> @arpit1503khanna is this resolved

I have asked aryansh sir , to let me know if the solution recommended is working or not.

**Pull request closed without merging**